src/extensions/core/load3d/Load3DConfiguration.ts

```diff
@@ -72,15 +73,21 @@ class Load3DConfiguration {
   private setupDefaultProperties() {
     const cameraType = this.load3d.loadNodeProperty(
       'Camera Type',
-      'perspective'
+      useSettingStore().get('Comfy.Load3D.CameraType')
```

Comment:
So we are allowing user configure the default setting when node is created. Can we better describe that in settings text?

Reply:
done